### ISSUE TITLE
Add support for CSV and other table's contents in Translation module

### ISSUE THREAD

**Issue by kavitharaju** [Bridgeconn/vachan-api#368]
Now the [translation module](https://api.vachanengine.org/docs#/Autographa-Project%20management/update_project_v2_autographa_projects_put ) can take in a USFM file or contents from bible tables on the server and use them as source for translation.

Implement support for uploading CSV(and other formats, if possible)  files and using those contents for translation.
Also, implement support for using text content from other kinds of tables like commentaries using(adding that support to) the [get-sentence API](https://api.vachanengine.org/docs#/Sources/extract_text_contents_v2_sources_get_sentence_get)

**Comment by kavitharaju** [comment]
We do have provision now, to upload content as list of sentences with sentence ids and optional surrogate ids. The client App can handle the file formats like CSV etc. and the fields they expect in it depending on use case(eg: Bible translation can expect verse references which will become surrogate ids and story translation can use a combination of para_nnumber and sentence number, as surrogate id,  which will help the UI to create proper drafts etc).

Basically, keeping the format generic at API side, expecting just list of sentences and allowing the clients to handle data formats and  customize it as per their needs by the use of appropriate surrogate_id values
